contrib/docker/Makefile

```diff
@@ -65,13 +76,30 @@ endif
 
 ifeq ("$(shell echo ${OS} | grep centos)","centos74")
     RUN_AS_USER_SCRIPT=${DOCKUSER_HOME}/ngraph-test/contrib/docker/run_as_centos_user.sh
-    DOCKERFILE=Dockerfile.ngraph.centos74
+    CPU_DOCKERFILE=Dockerfile.ngraph.centos74
```

Comment:
I am also a little confused as to why the Dockerfile name does not have the "_gpu" at the end, since you have created _gpu Dockerfiles for this commit?

Reply:
crlishka: possibly because the generically named Dockerfile.ngraph.centos74 is used for cpu and is assigned to CPU_Dockerfile, while if the Nvidia-smi command is available, then the GPU_Dockerfile variable is set to the 'generically' named docker file with the appended "_gpu"...see line 89 below...

Reply:
@raramer01 Yes you are correct.  Looking further, I see that GPU_DOCKERFILE is derived from CPU_DOCKERFILE a little later.  That clears my question up.